corehq/apps/cloudcare/templates/cloudcare/config.html

```diff
@@ -11,6 +11,13 @@
   {% registerurl "cloudcare_app_settings" domain %}
 
   <h2>{% trans 'Manage Web Apps Permissions' %}</h2>
+  <div class="alert alert-danger">
+    {% url "roles_and_permissions" domain as roles_url %}
+    {% blocktrans %}
+      This feature is deprecated. Access to specific web apps can now be managed via
+      <a target="_blank" href="{{ roles_url }}">Roles & Permissions</a>.
+    {% endblocktrans %}
```

Comment:
lol, yup there it is

Reply:
It's a good point that the flag itself could be more descriptive, updated in 7aa89a67b64